packages/integrations/mdx/src/rehype-optimize-static.ts

```diff
@@ -29,10 +34,11 @@ export function rehypeOptimizeStatic(options?: OptimizeOptions) {
 		for (const child of tree.children) {
 			if (child.type === 'mdxjsEsm' && exportConstComponentsRe.test(child.value)) {
 				// Try to loosely get the object property nodes
-				const objectPropertyNodes = child.data.estree.body[0]?.declarations?.[0]?.init?.properties;
+				const objectPropertyNodes =
+					child.data.estree?.body[0]?.declaration?.declarations?.[0]?.init?.properties;
```

Comment:
The previous access was incorrect and never actually worked 😅 I should look into properly typing this in the future.